lib/WebServer/ConfigWebServer.cpp

```diff
@@ -123,10 +123,9 @@ void ConfigWebServer::begin(Settings& settings) {
             }));
 
   server.on(F("/firmware"), HTTP_GET, authenticated([this]() {
-              server.send(200, APPLICATION_JSON,
-                          String(F("{\"version\": \"")) +
-                              String(QUOTE(FIRMWARE_VERSION)) +
-                              String(F("\"}")));
+              server.send_P(
+                  200, APPLICATION_JSON,
```

Comment:
send_P expects the content type as a PGM_P.

Reply:
We can change it again, but `send_P()` is esp specific and loading `char*` instead of `PGM_P` will not crash (only the opposite way).